lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php

```diff
@@ -720,6 +723,7 @@ protected function _unQuote($string)
      * Normalizes mixed positional-named bind to positional bind, and replaces named placeholders in query to
      * '?' placeholders.
      *
+     * @deprecated
```

Comment:
Could you add description why it became deprecated and what should be used instead?

Reply:
This function is never used. But it's protected so I can't just delete it

Reply:
Could you just add comment that it’s never used?
```suggestion
     * @deprecated not used anywhere 
```